sky/data/storage.py

```diff
@@ -45,6 +48,29 @@ class StorageMode(enum.Enum):
     COPY = 'COPY'
 
 
+def get_storetype_from_cloud(cloud: clouds.Cloud) -> StoreType:
+    if isinstance(cloud, clouds.AWS):
+        return StoreType.S3
+    elif isinstance(cloud, clouds.GCP):
+        return StoreType.GCS
+    elif isinstance(cloud, clouds.Azure):
+        return StoreType.AZURE
```

Comment:
omit? (and change L60 message)? otherwise, `add_store(StoreType.AZURE)` could've been called.